pkg/kubevirt/machine_server.go

```diff
@@ -100,6 +100,12 @@ func (p *MachinePlugin) DeleteMachine(ctx context.Context, req *driver.DeleteMac
 		return nil, util.PrepareErrorf(err, "Create machine %q failed on DecodeProviderSpecAndSecret", req.Machine.Name)
 	}
 
+	validationErrors := validation.ValidateKubevirtSecret(providerSpec, req.Secret)
```

Comment:
Could you please rename this method to reflect what it actually does, e.g. `ValidateKubevirtProviderSpecAndSecret`?